normandy/settings.py

```diff
@@ -120,5 +120,4 @@ class Production(Base):
 class Test(Base):
     SECRET_KEY = values.Value('not a secret')
     PROD_DETAILS_STORAGE = values.Value('product_details.storage.PDFileStorage')
-    DATABASES = values.DatabaseURLValue('sqlite://:memory:')
```

Comment:
py.test correctly creates test database(s) for us like `test_normandy`. Plus, we can't use sqlite since we have tied ourselves to Postgres.